client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.html

```diff
@@ -25,13 +36,11 @@ <h2 class="wildfire-count">
                 </button>
             </div>
             <div class="firecentre-filter">
-                <select appWFSingleSelect appWFPlaceholder="Select a fire centre..."
-                    [options]=""
-                    [selected]="">
+                <select appWFSingleSelect appWFPlaceholder="Select a fire centre..." [options]="" [selected]="">
```

Comment:
We have removed the Fire Centre filter from the UI. Not really a part of this ticket though so we can remove this as a separate task

Reply:
Yeah, I just grabbed the UI that Lucas leave me in another branch. Makes sense.